lib/uplink/project.go

```diff
@@ -150,6 +152,11 @@ func (p *Project) GetBucketInfo(ctx context.Context, bucket string) (b storj.Buc
 func (p *Project) OpenBucket(ctx context.Context, bucketName string, access *EncryptionAccess) (b *Bucket, err error) {
 	defer mon.Task()(&ctx)(&err)
 
+	attribution, err := p.CheckBucketAttribution(ctx, bucketName)
+	if err != nil {
+		zap.S().Warn("Attribution error")
```

Comment:
It's not an attribution error, just no attribution was performed.

Reply:
updated the error